app/views/papers/index.html.erb

```diff
@@ -1,29 +1,65 @@
+<%= stylesheet_link_tag 'papers', media: 'all', 'data-turbolinks-track': 'reload' %>
+
 <p id="notice"><%= notice %></p>
 
-<h1>Papers</h1>
+<p id="papers">Papers</p><br>
+
+<%= form_tag papers_path, { :method => 'get', class: "container", role: "search" }  do %>
+	<div class="row">
+      <div class="col-md-6">
+				<h4>Title</h4>
+          <div id="custom-search-input">
+              <div class="input-group col-md-12">
+                  <div class="form-group">
+                     <%= text_field_tag :search, "", class: "form-control", placeholder: "Title" %>
+                   </div>
+                    <span class="input-group-btn">
+                      <button class="btn btn-info btn-lg" type="button">
+                          <i class="glyphicon glyphicon-search"></i>
+                      </button>
+                    </span>
+                </div>
+            </div>
+        </div>
+	</div>
+<% end %><br><br>
 
-<table>
+<div style="margin-left: 1095px">
```

Comment:
Avoid inline CSS: these properties should be in classes or IDs defined in your .scss files